README.md

```diff
@@ -0,0 +1,32 @@
+# OpenAstronomy conda channel
+
+## Installing packages
+
+To add this channel to your list of default conda channels:
+`conda config --add channels openastronomy`
+
+To install a single package from this channel:
+`conda install -c openastronomy <package>`
+
+## List of packages
+
+The list of packages on this channel is at
+https://anaconda.org/openastronomy/packages
+
+It includes all of the packages on these conda channels:
+
++ `astropy`: channel for [astropy](http://astropy.org) [affiliated packages](http://astropy.org/affiliated)
+
+## How can I add my package?
```

Comment:
Done - see #10.

One item that's specifically about the README, though: do you think it's reasonable to explicitly say that a way to get on this list (maybe even a "recommended" way) is to be astropy-affiliated?  If so, I can make a PR to do that.